jenkins/deduper.py

```diff
@@ -41,8 +41,8 @@ def _aborted_description(current_build_id, pr):
 
         :Returns: A description (string)
         """
-        return ("[PR #{}] Build automatically aborted because"
-                " there is a newer build for the same PR. See build"
+        return ("[PR #{}] You have a newer build running for this PR."
+                " Go see its results instead! Build"
```

Comment:
Just getitng to this PR. But I'd only nit and say 
` Go see its results instead! See build`

So, the 'go to this build#' is still a directive. A link sounds good..perhaps worth a look.